### PR TITLE
Adding copy to clipboard for containerImages

### PR DESCRIPTION
**Pull request by galletti94**
Screenshot from 2019-04-10 17-04-51
Screenshot from 2019-04-10 17-04-53

### CODE REVIEW

**Comment by tlwu2013** [comment]
Hey @galletti94 , thanks for your prompt support!

Just one minor thing, can you help add a little `margin-left` on the "copy icon" so that the icon is not directly attached to the link?
(similar to the "external icon" above)
screen-mocked

Thanks a lot Lance!

File: frontend/src/components/OperatorSidePanel.js [hunk 1]
```diff
@@ -166,39 +136,111 @@ const OperatorSidePanel = ({ operator, showInstall, updateChannel, updateVersion
...
+
+    const imageLink = containerImage ? <Tooltip content={tooltipContent} styles={tooltipOverrides}>
+      <a
+        href="#"
+        onClick={e => this.copyToClipboard(e, containerImage)}
+        onMouseEnter={this.onCopyEnter}
+      >
+        {containerImage}
+        <Icon type="fa" name="clipboard" />
+      </a>
```
Comment (jeff-phillips-18) [inline frontend/src/components/OperatorSidePanel.js]: Only the Icon should be the clickable anchor, the text should just be in a normal span.

Comment (galletti94) [reply frontend/src/components/OperatorSidePanel.js]: Ok will do

**Comment by galletti94** [comment]
Screenshot from 2019-04-11 10-23-52
Screenshot from 2019-04-11 10-23-54
@tlwu2013 @jeff-phillips-18

**Pull request merged**